pkg/agent/controller/networkpolicy/packetin.go

```diff
@@ -78,13 +87,40 @@ func initLogger() error {
 	return nil
 }
 
-// HandlePacketIn is the packetin handler registered to openflow by Antrea network policy agent controller.
-// Retrieves information from openflow reg, controller cache, packetin packet to log.
+// HandlePacketIn is the packetin handler registered to openflow by Antrea network
+// policy agent controller. It choose different operations based on custom reasons
+// in the received packet from OVS.
```

Comment:
s/It choose different operations based on custom reasons in the received packet from OVS./It performs the appropriate operations based on which bits are set in the "custom reasons" field of the packet received from OVS.

Reply:
Done.